engine/sound/src/test/test_sound.cpp

```diff
@@ -398,6 +402,60 @@ void DeviceLoopbackStop(dmSound::HDevice device)
 
 }
 
+#if !defined(GITHUB_CI) || (defined(GITHUB_CI) && !(defined(WIN32) || defined(__MACH__)))
+TEST_P(dmSoundTestLoopingTest, Loopcount)
+{
+    TestParams params = GetParam();
+    dmSound::Result r;
+    dmSound::HSoundData sd = 0;
+    dmSound::NewSoundData(params.m_Sound, params.m_SoundSize, params.m_Type, &sd, 1234);
+
+    printf("tone: %d, rate: %d, frames: %d\n", params.m_ToneRate, params.m_MixRate, params.m_FrameCount);
+
+    dmSound::HSoundInstance instance = 0;
+    r = dmSound::NewSoundInstance(sd, &instance);
+    ASSERT_EQ(dmSound::RESULT_OK, r);
+    ASSERT_NE((dmSound::HSoundInstance) 0, instance);
+
+    int8_t loopcount = 5;
+    r = dmSound::SetLooping(instance, 1, loopcount);
+    ASSERT_EQ(dmSound::RESULT_OK, r);
+
+
+    r = dmSound::Play(instance);
+    ASSERT_EQ(dmSound::RESULT_OK, r);
+    printf("Playing: %d\n", dmSound::IsPlaying(instance));
+
+    do {
+        r = dmSound::Update();
+    } while (dmSound::IsPlaying(instance));
+    r = dmSound::DeleteSoundInstance(instance);
+    ASSERT_EQ(dmSound::RESULT_OK, r);
+
+    // This is set up by observation: after first iteration, m_Time is 164.
+    // For each loop done afterwards, another 172 is added. 
+    ASSERT_EQ(g_LoopbackDevice->m_Time, 164 + 172*loopcount); 
```

Comment:
I don't currently have a better idea I'm afraid.
It's a bit disturbing though that the multiples differ (164 != 172)

Reply:
Maybe, we discovered something :-). Shall i open an issue for that, to further investigate if the time comes ?

Reply:
Nah, I think we'll just have to live with it for now.
I think it'll be fixed/resolved once we get cursor support or something similar, where we need better accuracy.